### PR TITLE
rewrite core to better functional

### PR DESCRIPTION
**Pull request by bohdanlisovskyi**

### CODE REVIEW

**Review by YAWAL — APPROVED** [review]

File: server/main.go [hunk 1]
```diff
@@ -11,7 +11,7 @@ func main() {
 	loger.Log.Infof("Server run")
 	r := mux.NewRouter()
 	r.HandleFunc("/message", core.MessageHandler) // listen message
-	err := http.ListenAndServe(":3002", r)
+	err := http.ListenAndServe(":3006", r)
```
Comment (8tomat8) [inline server/main.go]: Move `host` and `port` to config file.

File: server/message/core-message.go [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+func SendMessage(message *messageService.Message, messageType int) messageService.Message {
+
+	byteMessage, err := messageService.MarshalMessage(message)
+	if err != nil {
+		loger.Log.Errorf("Marshal message error: ", err.Error())
+		return messageService.Message {
+			Header:messageService.MessageHeader{
+				Type_: MessageType,
+				Command: "Marshal message error",
```
Comment (8tomat8) [inline server/message/core-message.go]: `Commands` could be also moved to constants

File: server/message/core-message.go [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+	}
+
+	return writeMsg(byteMessage, message, messageType) //I send this text []byte to receiver
+}
+
+func writeMsg(text []byte, message *messageService.Message, messageType int) messageService.Message {
+	msgBody := messageService.MessageBody{}
+	err := json.Unmarshal(message.Body, &msgBody)
+	if err != nil {
+		loger.Log.Errorf("Unmarshal message error: ", err.Error())
```
Comment (8tomat8) [inline server/message/core-message.go]: This row should look like this:
```loger.Log.Errorf("Unmarshal message error: %s", err)```
or:
```loger.Log.Error("Unmarshal message error: ", err)```
or:
```loger.Log.Error("Unmarshal message error: ", err.Error())```

File: server/modules/modules.go [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+	if err.Header.Command != "Ok" {
+		err := conn.WriteMessage(messageType, byteError)
+
+		if err != nil {
+			loger.Log.Errorf("Write Message Error")
+		}
+	}
+}
+
+func Register(message *messageService.Message, conn *websocket.Conn) {
```
Comment (8tomat8) [inline server/modules/modules.go:34]: Please syncronize with @Greckas  about `Register` and `Auth` methods

**Review by 8tomat8 — APPROVED** [review]

File: server/modules/modules.go (hunk 4 above)
Comment (bohdanlisovskyi) [reply server/modules/modules.go:34]: He also rewrite this function like he need, I wrote this for example!

**Pull request merged**